cucumber/api/features/step_definitions/audit_steps.rb

```diff
@@ -47,6 +53,15 @@ def matcher val
     else match val
     end
   end
+
+  def normalized_to_log(message)
```

Comment:
It would be great to add some short description to what the format should look like before and after

Reply:
Will do